packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -148,6 +156,12 @@ export class VaTextInput {
     }
   };
 
+  connectedCallback() {
+    i18next.on('languageChanged', () => {
+      forceUpdate(this.el);
+    });
+  };
```

Comment:
Lets have a `i18next.off('languageChanged')` in a `disconnectedCallback` as well.

Reply:
added